test/conformance/network_policy_v2_test.go

```diff
@@ -50,16 +54,50 @@ func TestNetworkPolicyV2Conformance(t *testing.T) {
 	t.Log("Starting the network policy V2 conformance test suite")
 	// Depending on the tests some of them take longer and end up timing
 	// out on context, let's bump the GetTimeout to 300 seconds here.
-	cSuite := suite.New(suite.Options{
-		Client:                     client,
-		ClientSet:                  clientset,
-		KubeConfig:                 *cfg,
-		Debug:                      showDebug,
-		CleanupBaseResources:       shouldCleanup,
-		EnableAllSupportedFeatures: enableAllSupportedFeatures,
-		BaseManifests:              conformanceTestsBaseManifests,
-		TimeoutConfig:              netpolv1config.TimeoutConfig{GetTimeout: 300 * time.Second},
-	})
+	profiles := sets.Set[suite.ConformanceProfileName]{}
+	profiles.Insert(suite.ConformanceProfileName(suite.SupportAdminNetworkPolicy))
+	profiles.Insert(suite.ConformanceProfileName(suite.SupportBaselineAdminNetworkPolicy))
+	cSuite, err := suite.NewConformanceProfileTestSuite(
+		suite.ConformanceProfileOptions{
+			Options: suite.Options{
+				Client:                     client,
+				ClientSet:                  clientset,
+				KubeConfig:                 *cfg,
+				Debug:                      showDebug,
+				CleanupBaseResources:       shouldCleanup,
+				EnableAllSupportedFeatures: enableAllSupportedFeatures,
+				BaseManifests:              conformanceTestsBaseManifests,
+				TimeoutConfig:              netpolv1config.TimeoutConfig{GetTimeout: 300 * time.Second},
+			},
+			Implementation: confv1a1.Implementation{
+				Organization: "ovn-org",
+				Project: "ovn-kubernetes",
+				URL: "https://github.com/ovn-org/ovn-kubernetes",
+				Version: "v0.1.2",
+				Contact: []string{"@tssurya"},
+				AdditionalInformation: "https://github.com/ovn-org/ovn-kubernetes/blob/e0d2ab8afd043a03c6e6b2935a0d773810dad0f7/test/conformance/network_policy_v2_test.go#L50",
+			},
+			ConformanceProfiles: profiles,
+		})
+	if err != nil {
+		t.Fatalf("error creating conformance test suite: %v", err)
+	}
 	cSuite.Setup(t)
 	cSuite.Run(t, tests.ConformanceTests)
+	const reportFileName = "ovn-kubernetes-anp-test-report.yaml"
+	t.Log("saving the network policy conformance test report to file:", reportFileName)
+	report, err := cSuite.Report()
+	if err != nil {
+		t.Fatalf("error generating conformance profile report: %v", err)
+	}
+	t.Log("Printing report...", report)
+	rawReport, err := yaml.Marshal(report)
+	if err != nil {
+		t.Fatalf("error marshalling conformance profile report: %v", err)
+	}
+	t.Log("Printing raw report...", rawReport)
+	err = os.WriteFile("../../"+reportFileName, rawReport, 0o600)
```

Comment:
` 0o600` doesnt seem right - shouldnt it be only integers

Reply:
it is an integer, but in base 8 :)

Reply:
ah so `o` represents base 8.. news to me.
I thought hex maybe ok here but didnt realise `o` is base 8. Thanks!

Reply:
the o stands for octal :)